apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -66,12 +67,12 @@ const ProgressBubble = React.createClass({
       href = url + location.search;
     }
 
+    const tooltipId = _.uniqueId();
```

Comment:
This was done because in storybooks we end up with multiple bubbles that have the same URL (even though I think this should never happen on a real page). I alternatively could have tried doing the work to make our storybooks always provide unique urls, but this approach seems more resilient.